octodns_powerdns/__init__.py

```diff
@@ -364,7 +384,7 @@ def populate(self, zone, target=False, lenient=False):
             exists = True
             for rrset in resp.json()['rrsets']:
                 _type = rrset['type']
-                if _type == 'SOA':
+                if _type in self.SUPPORTS:
```

Comment:
Doh, I mis-read the direction of the check 

```suggestion
                if _type not in self.SUPPORTS:
```